packages/datetime/src/dateRangeInput.tsx

```diff
@@ -46,22 +69,82 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, {}>
         },
     };
 
+    public constructor(props: IDateRangeInputProps, context?: any) {
+        super(props, context);
+        this.state = {
+            isOpen: false,
+            selectedEnd: moment(null),
+            selectedStart: moment(null),
+        };
+    }
+
     public render() {
+        const popoverContent = (
+            <DateRangePicker
+                onChange={this.handleDateRangePickerChange}
+                maxDate={this.props.maxDate}
+                minDate={this.props.minDate}
```

Comment:
Do you foresee more and more of the `DateRangePicker` options needing to be passed through, and maybe using some filtered down variant of `{ dateRangePickerProps }` similar to how you're doing the input props?